public/media_engagement/worldbank_plr_sms_traffic.html

```diff
@@ -48,7 +48,7 @@
                         <a href="#" class="nav-link border-top border-right active-link dropdown-toggle" data-toggle="dropdown">TRAFFIC</a> 
                         <!-- DROPDOWN MENU -->
                         <div class="dropdown-menu">
-                            <a class="dropdown-item active-item" href="https://avf-dashboards.firebaseapp.com/media_engagement/imaqal_media_engagement.html">IMAQAL</a> 
+                            <a class="dropdown-item" href="https://avf-dashboards.firebaseapp.com/media_engagement/imaqal_media_engagement.html">IMAQAL</a> 
```

Comment:
Please add all the active project links (Imaqal + Worldbank for now)

Reply:
I think it would be better if we remove the displayed project from the dropdown menu and include it when another project is displayed since the link will just redirect to the same page.

Reply:
Ahh I see. Thanks

Reply:
add the change to imaqal_sms_traffic.html too and request re-review.

Reply:
done